docs/tutorial/index.md

```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
+   dich durch das Aufsetzen deiner Entwicklungsumgebung.
+1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Der Beginn neuer
+   Projekte, das Entwickeln und Veröffentlichen von Webseiten.
+2. [Einführung in die Nutzung von CSS in Gatsby](/tutorial/part-two/): Entdecke
+   Biblitheken wie Typography.js und CSS Module.
+3. [Bauen von verschachtelten Layouts in Gatsby](/tutorial/part-three/):
+   Layouts sind Sektionen deiner Seite, die über verschiedene Seiten wieder
+   benutzt werden, wie Kopf- und Fußzeilen.
 
-## Intermediate tutorials
+## Fortgeschrittene Tutorials
 
-In these intermediate tutorials, you'll learn how to pull data from almost anywhere into your Gatsby site with GraphQL.
+In diesen fortgeschrittenen Tutorials, wirst du lernen wie du Daten von nahezu
+überall mit Hilfe von GraphQL in deine Gatsby Seite integrieren kannst.
 
-4.  [Querying for data in a blog](/tutorial/part-four/): Create a blog and use a GraphQL query to pull your site title into the blog header.
-5.  [Source plugins and rendering queried data](/tutorial/part-five/): Use a source plugin to pull Markdown blog posts into your site and create an index page with a list of blog posts.
-6.  [Transformer plugins](/tutorial/part-six/): Use a transformer plugin to transform your Markdown blog posts into a form the blog can render.
-7.  [Programmatically create pages from data](/tutorial/part-seven/): Learn how to programmatically create a set of pages for your blog posts.
-8.  [Preparing a site to go live](/tutorial/part-eight/): Learn how to audit your site for performance and best practices for accessibility, SEO, and more.
+4. [Abfragen von Daten in einem Blog](/tutorial/part-four/): Erstellung eines
+   Blogs und Nutzung einer GraphQL Anfrage um deinen Webseiten Titel in die
+   Kopfzeile des Blogs zu ziehen.
+5. [Datenquellen Plugins und Darstellung von angefragten Daten](/tutorial/part-five/):
+   Nutzen eines Datenquellen Plugins um Markdown Blog Beiträge in deine Seite
+   zu ziehen und Erstellung einer Index Seite mit der Liste aller Blog Beiträge.
+6. [Umformende Plugins](/tutorials/part-six/): Nutzung eines umformenden
+   Plugins um Markdown Blog Beiträge in eine Form umzuwandeln, die der Blog
+   darstellen kann.
+7. [Programatisches Erstellen von Seiten aus Daten](/tutorial/part-seven/):
+   Lerne wie Seiten für deinen Blog programatisch erstellt werden können.
+8. [Vorbereiten einer Seite zum Veröffentlichen](/tutorial/part-eight/):
+   Lerne wie du deine Seite auf Performance und empfohlene Vorgehensweisen
+   für Barrierefreiheit, SEO und mehr prüfen kannst.
 
-## Theme tutorials
+## Themen Tutorials
 
-Gatsby themes abstract all of your default configuration (shared functionality, data sourcing, design) out of your site and into an installable package.
+Gatsby Themen abstrahieren alle Standardeinstellungen (geteilte Funktionalität,
+Datenquellen, Design) aus deiner Seite und in ein installierbares Paket.
```

Comment:
Despite the "and" is present in english, I'd remove "und" here. Seems a little bit out of place here, if you try to read the whole sentence.

Reply:
Hopefully done in #10